src/html/index.html

```diff
@@ -5,29 +5,27 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Document</title>
   <link rel="stylesheet" href="/dist/css/style.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.2/css/all.min.css" integrity="sha512-z3gLpd7yknf1YoNbCzqRKc4qyor8gaKU1qmn+CShxbuBusANI9QpRohGBreCFkKxLhei6S9CQXFEbbKuqLg0DA==" crossorigin="anonymous" referrerpolicy="no-referrer" />
 </head>
 <body>
-  <header>
-    <nav>
+  <header >
+    <nav class="navbar">
       <div>
-          <img>
-      </div>
-      <ul>
-          <li><a href="#">Home</a></li>
-          <li><a href="#">News</a></li>
-          <li><a href="#">About</a></li>
-      </ul>
-      <div>
-        <ul>
-          <li><a href="#">For Employers</a></li>
-          <li><a href="#">For Headhunters</a></li>
-        </ul>
-          <div>
-            <button></button>
-            <button></button>
-          </div>
+          <img src="../images/itapplogo_color.webp" >
       </div>
+        <button class="toggle-button" onclick="handleNavOpen()">
+          <i class="fa-solid fa-bars fa-2xl menu-icon"></i>
+        </button>
   </nav>
+  <div class="dropdown-menu dropdown-menu-off">
```

Comment:
Please can you have a look  https://trello.com/c/pWf2vVg7/16-theory-css-architecture